src/views/search/parts/tag-folder-row.tsx

```diff
@@ -88,7 +90,9 @@ const TagFolderRow: FC<TagFolderRowProps> = ({ compProps }): ReactElement => {
 					avatarBackground: getFolderIconColor(folderDestination),
 					avatarIcon: 'FolderOutline',
 					isQueryFilter: true,
-					value: `in:"${folderDestination?.absFolderPath}"`
+					value: folderDestination.id.includes(':')
```

Comment:
In order to centralise the logic in less places I suggest to use the _isSharedAccountFolder_ function (inside _helpers/folders.ts_) instead of the direct search for the colons